app/renderer/components/styles/tab.js

```diff
@@ -89,18 +86,19 @@ const styles = StyleSheet.create({
   },
 
   isPinned: {
-    paddingLeft: globalStyles.spacing.defaultIconPadding,
-    paddingRight: globalStyles.spacing.defaultIconPadding
+    padding: 0,
+    width: `calc(${globalStyles.spacing.tabsToolbarHeight} * 1.1)`,
```

Comment:
`26px * 1.1` is nearly equal to the pinned tab width where the padding is increased from 2px to 4px.

Reply:
this seems fragile to me. What if at a given point we decide to update tabs toolbar height?

pinned tabs sizing are also are not used anywhere else so I won't mind having the size explicitly described here.